src/container-manager/index.ts

```diff
@@ -27,97 +26,133 @@ export default class ContainerManager {
         return this.explorerEnvironment;
     }
 
-    public async startNotifier(instanceNumber: number): Promise<StartedGenericContainer> {
-        throw new Error("NOT_IMPLEMENTED");
+    public async startNotifier(
+        instanceNumber: number
+    ): Promise<StartedGenericContainer> {
+        throw new Error('NOT_IMPLEMENTED');
     }
 
-    private async startDockerComposeModule(buildContext: string): Promise<StartedDockerComposeEnvironment> {
+    private async startDockerComposeModule(
+        buildContext: string
+    ): Promise<StartedDockerComposeEnvironment> {
         return await new DockerComposeEnvironment(
             buildContext,
             `docker-compose.yml`
         ).up();
     }
 
     public getContainer(
-        containerName: 'rsk-node' | 'lumino-explorer' | 'mongo-db' | 'lumino-node' | 'boot-node',
+        containerName:
+            | 'rsk-node'
+            | 'lumino-explorer'
+            | 'mongo-db'
+            | 'lumino-node'
+            | 'boot-node',
         luminoNodeName?: string,
-        scaleNumber?: number): StartedGenericContainer | StartedTestContainer | undefined {
+        scaleNumber?: number
+    ): StartedGenericContainer | StartedTestContainer | undefined {
         switch (containerName) {
             case 'rsk-node':
                 return this.rskNodeEnvironment?.getContainer(containerName);
             case 'lumino-explorer':
             case 'mongo-db':
                 return this.explorerEnvironment?.getContainer(containerName);
             case 'lumino-node':
-                return this.luminoNodes.find(node => node.name === luminoNodeName)?.container;
+                return this.luminoNodes.find(
+                    (node) => node.name === luminoNodeName
+                )?.container;
             case 'boot-node':
                 return this.rifCommsBootNode;
         }
     }
 
     public async startupRifCommunicationsBootNode(): Promise<void> {
         console.debug('Creating rif-comms boot node...');
-        let rifCommsBootNodeContainer: GenericContainer = new GenericContainer('lumino-testing-rif-comms-boot-node-image:latest');
-        rifCommsBootNodeContainer = rifCommsBootNodeContainer.withNetworkMode(DOCKER_NETWORK_NAME).withName('lumino-testing-rif-comms-boot-node');
+        let rifCommsBootNodeContainer: GenericContainer = new GenericContainer(
+            'lumino-testing-rif-comms-boot-node-image:latest'
+        );
+        rifCommsBootNodeContainer = rifCommsBootNodeContainer
+            .withNetworkMode(DOCKER_NETWORK_NAME)
+            .withName('lumino-testing-rif-comms-boot-node');
         const rifCommsBootNodeStartedContainer = await rifCommsBootNodeContainer.start();
         await waitForHealthCheck({
             command: 'healthCheck',
             expectedResult: 'NODE ONLINE',
-            container: rifCommsBootNodeStartedContainer
+            container: rifCommsBootNodeStartedContainer,
```

Comment:
Check if you can remove this rule, trailing comma is not necessary and it makes the code dirty

Reply:
i don't agree, let's ask @mauricioirace what he thinks

Reply:
Actually, I think is not dirty, and it's easier to type afterwards or change order of the parameters. So, it's ok to me.

Reply:
Ok, so you are ok to put trailing comma always then, because with this rule if you forgot to put the trailing comma it will show you an error, even when the object is already fulfilled, i think is a dumb rule sorry the expression but anyway if you want it go ahead.

Reply:
what rule? I just see a comma added. I think it's ok to add a comma, but I don't think it should be mandatory

Reply:
it's trailing comma for linter and prettier plugin, correct me if i'm wrong @mortelli

Reply:
there's no error shown. this is an automatic formatter.